include/deal.II/distributed/shared_tria.h

```diff
@@ -302,6 +302,8 @@ namespace parallel
        */
       void partition();
 
+    private:
```

Comment:
do you need those other flags as protected as well in derived class? If no, move `private: right under `update_number_cache`.

Reply:
I wonder if those const members are useful to inherit (?)

Reply:
if you don't need it right now, then don't move them in `protected`.

Reply:
Yes, I agree -- only mark the one function as `protected` that you want to mark that way, but nothing else.